components/Stripe/CardStatusSwitchWidget.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+
+function CardStatusSwitchWidget({ cardId, cardStatus }) {
+  const [showModal, setShowModal] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+  const [error, setError] = useState(false);
+  const [errorText, setErrorText] = useState('');
+  return (
+    <div>
+      {cardStatus != "canceled" ?
+      <div>
+        {error ? (
+          <div className="p-2 text-center ">
+            <p className="text-red-600">{errorText}</p>
+          </div>
+        ) : null}
+
+        <button
+          id="switch-card-status-btn"
+          type="button"
+          className="inline-flex items-center px-10 py-2.5 border border-transparent shadow-sm text-xs md:text-sm font-medium rounded-md text-white bg-accent-color hover:bg-accent-color-light ml-2 mr-2"
+          onClick={async (e) => {
```

Comment:
lets move this handler above the JSX so it's easier to read.

Reply:
Fixed on commit [779b1b0](https://github.com/stripe-samples/treasury-nextjs/pull/19/commits/779b1b01fe53154e3ad6d07714f1c83c51607c17)